src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PrisonResourceTest.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.junit.jupiter.api.Test
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.MediaType
+import uk.gov.justice.hmpps.prison.api.model.SentenceSummary
+
+class PrisonResourceTest : ResourceTest() {
+
+  @Test
+  fun testPrisonBookingSummaryReturnsListResponseReturnsSentenceSummaryList() {
+    val establishment = "LEI"
+
+    webTestClient.get()
+      .uri("/api/prison/{establishment}/booking/latest/sentence-summary", establishment)
+      .headers(
+        setAuthorisation(
+          listOf("ROLE_RELEASE_DATE_MANUAL_COMPARER"),
+        ),
+      )
+      .header("Content-Type", MediaType.APPLICATION_JSON_VALUE)
+      .accept(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus().isOk
+      .expectBodyList(object : ParameterizedTypeReference<SentenceSummary>() {})
```

Comment:
Not interested in the contents of the response? Normally we have a peek in the json using the likes of
```
.expectBody()
.jsonPath("$[0].prisonerNumber").isEqualTo("A1234AA")
.jsonPath("$[0].latestPrisonTerm").isEqualTo("TERM")
```

Reply:
Mike I've addressed this one :)